### PR TITLE
Add boskos leases for the 2nd azure subscription

### PR DESCRIPTION
**Pull request by droslean**
/cc @openshift/test-platform 

Signed-off-by: Nikolaos Moraitis <nmoraiti@redhat.com>

### CODE REVIEW

**Review by petr-muller — COMMENTED** [review]
You also need to add it to the reaper: https://github.com/openshift/release/pull/21542

File: core-services/prow/02_config/generate-boskos.py [hunk 1]
```diff
@@ -31,6 +31,12 @@
         'eastus2': 8,
         'westus': 8
     },
+    'azure-2-quota-slice': {
+        'centralus': 33,
+        'eastus': 8,
+        'eastus2': 8,
+        'westus': 8
+    },
```
Comment (petr-muller) [inline core-services/prow/02_config/generate-boskos.py:39]: How were these numbers computed?

Comment (droslean) [reply core-services/prow/02_config/generate-boskos.py:39]: I used the same with `azure4`.

Comment (petr-muller) [reply core-services/prow/02_config/generate-boskos.py:39]: Which may not be correct. How do we know we have enough quota in the second account?

Comment (petr-muller) [reply core-services/prow/02_config/generate-boskos.py:39]: Do we have a similar spreadsheet like we used for AWS2 and GCP2? If yes, have you checked the quotas? If no, can we get one?

Comment (droslean) [reply core-services/prow/02_config/generate-boskos.py:39]: The 2nd subscription has exactly the same quotas as the 1st one.

Comment (droslean) [reply core-services/prow/02_config/generate-boskos.py:39]: Let me correct me here. The 2nd subscription with specific quotas have been requested in https://issues.redhat.com/browse/DPP-7124. Let me /hold this and ask James for the speadsheet so we can calculate the boskos leases with an exact number.

Comment (droslean) [reply core-services/prow/02_config/generate-boskos.py:39]: @wking How can we calculate this like you did in https://github.com/openshift/release/pull/12840 ?

**Comment by droslean** [comment]
@wking Can you take a look on this?

File: core-services/prow/02_config/generate-boskos.py (hunk 1 above)
Comment (wking) [reply core-services/prow/02_config/generate-boskos.py:39]: The commit/PR message in #12840 has the method I used previously.  Is there a particular step that wasn't clear?

Comment (droslean) [reply core-services/prow/02_config/generate-boskos.py:39]: Ok so we are

- Central US:
    - 1200 standardDSv3Family / 40 = 30 clusters
    - 600 LowPriorityCores / 18 vCPUs per spot test  = 33 clusters
    - 100 Public IPs / 3 = 33 clusters
   
- eastus/eastus2/westus
  - 350 / 40 = 8 clusters
  -  350 LowPriorityCores / 18 = 19 clusters
  - 100 Public IPs / 3 = 33 clusters


So we are good to merge :)

**Review by petr-muller — APPROVED** [review]

**Pull request merged**